### ISSUE TITLE
More info on hosts outstanding

### ISSUE THREAD

**Issue by cclements** [byt3bl33d3r/CrackMapExec#160]
Often times when running the mimikatz module on a subnet, I'll have several stragglers, e.g.:

```
MIMIKATZ                                       [*] Waiting on 4 host(s)
MIMIKATZ                                       [*] Waiting on 4 host(s)
```

These stragglers never seem to finish, and there doesn't seem to be a straightforward way to get more information on them (hostname/IP, etc.) without digging through and correlating log entries.

Not a big deal, but it would be useful to have this info echoed back with the "Waiting" message or other such means such as an exit summary.

**Comment by byt3bl33d3r** [comment]
good feedback, will see what I can do.

Cheers

**Comment by kildonan5** [comment]
sorry if this isnt the right place for it, but i dont think it warrants its own issue. if i run mimikatz module on a host that doesnt have powershell (i.e. a windows 2003 box) CME just looops on waiting on 1 hosts. as a suggestion, maybe theres a way to detect if powershell is available, and error out if not?

In any case, this tool CME absolutely rocks.  should save me loads of time. thanks.

**Comment by tanujain** [comment]
In my case I do have powershell installed on the system I am trying to exploit but still I see the same error message. Waiting on 1 host.  Any suggestion on what could be the reason and how to fix it?

**Comment by mpgn** [comment]
Probably flaged by defender, we can't do much since we cannot know if the remote script is blocked by AV or just running. Make sure your enumeration on the target is complete before using a Module.
